modules/tinymce/src/core/test/ts/browser/init/InitDocumentWriteTest.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { SelectorFind } from '@ephox/sugar';
+import { TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.init.InitDocumentWriteTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    init_content_sync: true
+  }, [], true);
+
+  it('TINY-9818: Should initialize the editor', () => {
```

Comment:
Should we use `UiFinder` from agar here to be more consistent with other test files?

Reply:
Right somehow my brain never things of UiFinder when finding non UI things. Should really be called ElementFinder.